text/0000-pack-cache-options.md

```diff
@@ -0,0 +1,222 @@
+# Meta
+[meta]: #meta
+- Name: Pack cache options
+- Start Date: (fill in today's date: YYYY-MM-DD)
+- Author(s): [@jromero](https://github.com/jromero), [@dwillist](https://github.com/dwillist)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+<!-- One paragraph explanation of the feature. -->
+
+Allow users to configure the various cache options.
+
+# Definitions
+[definitions]: #definitions
+
+<!-- Make a list of the definitions that may be useful for those reviewing. Include phrases and words that buildpack authors or other interested parties may not be familiar with. -->
+
+#### General
+
+* **bind mounts** - Directories mounted from the docker host system onto the running container.
+* **named volumes** - Volumes that are referenced by name within docker and not directly associated with the host file system.
+* **lifecycle** - Component that executes in a container running all the necessary buildpacks and producing the final app image.
+
+#### Cache Types
+* **build cache** - Cache container `build` layers. That is layers that were created by buildpacks and marked as `build=true` and `cache=true`.
```

Comment:
```suggestion
* **build cache** - Cache containing layers that should be restored on subsequent builds. That is layers that were created by buildpacks and marked as `cache=true`.
```
Layers don't need to be `build=true` to be cached.

Reply:
That's interesting and honestly confusing but I'll take your word for it. :)